pkgs/applications/virtualization/virtualbox/guest-additions/strlcpy.patch

```diff
@@ -0,0 +1,116 @@
+# https://www.virtualbox.org//changeset/102989/vbox
```

Comment:
I find these source references easier to find, if they are placed where the patch is referenced rather than inside the patch itself.

It would also be nice to fetch these using `fetchpatch`, but the vbox SVN UI doesn't seem to allow this. :hankey: